src/component/mxgraph/BpmnRenderer.ts

```diff
@@ -79,46 +79,46 @@ export class BpmnRenderer {
   }
 
   private insertEdges(edges: Edge[]): void {
-    edges.forEach(edge => {
-      const bpmnElement = edge.bpmnElement;
+    edges.forEach(bpmnEdge => {
+      const bpmnElement = bpmnEdge.bpmnElement;
```

Comment:
**suggestion:** 
Reading `bpmnEdge.bpmnElement` is not easy for me. The abundance of `bpmn` is a bit overwhelming! 
How about using something like `internalEdge.bpmnElement` instead? 😄

Reply:
Let's go with `internalEdge`.
However, I won't update the edges parameter: this is how it is seen outside of the method, and I want to keep as it.
The `internalEdge` is for the loop only.

Does this work for you @csouchet?

Reply:
It's ok for me :slightly\_smiling\_face:

Reply:
done in 14454c6e